### ISSUE TITLE
Modular Powersuits Terminal Glasses Module

### ISSUE THREAD

**Issue by 0tobsam0** [OpenMods/OpenPeripheral-Integration#32]
In case someone besides me is looking for the Modular Powersuits Terminal Glasses Module that once was part of OpenPeripherals - it's gone, stop looking. The MPS integration seemingly had a short lifespan between 2013-06-09 and 2014-01-01 (according to the actual GitHub commit history). That last build with the integration was for MC version 1.6.4, so I guess a 1.7.10 compatible version of the MPS integration simply doesn't exist. The removal was part of some refactoring and splitting OpenPeripherals into it's Core, Addons and Integration parts.

If any of the devs read this - I would love to see a comeback of that integration. The terminal glasses are a wonderful and powerful addition to Computercraft, but not being able to combine them with armor is making them pretty much useless in most cases.

Another simple, but perfect solution to this problem would be to allow the terminal glasses to be equipped in the Tinker's Construct mask slot or the Baubles amulet slot that exists in most modpacks and rarely has a use anyway. That way you could combine it with any helmet you wish.

**Comment by bailwillharr** [comment]
Yes! This would be fantastic. It is quite frustrating that they removed this feature. It was very useful back in 1.6.4 Tekkit.
